tools/docker/tests/test_flows.sh

```diff
@@ -49,12 +49,13 @@ test_channel_selection() {
     return 1
 }
 
-test_ap_capability_info_reporting() {
-    #TODO: Implement
-    return 1
+test_ap_capability_query() {
+    dbg "sending ap capability query message"
+    eval send_bml_command "bml_trigger_ap_capability" $AL_MAC $redirect
```

Comment:
I agree with you Arnout.
And it is a really small change too. Just dropping all the commits related to bml, move the creation of the CMDU to wfa-ca, and update the test flow to send "bml_wfa_ca_controller <command>".

And on the way, do the same to topology query 😈

Reply:
@arnout I agree - but I asked Gal to do this since its better than nothing and we're not in the phase that we can use `bml_wfa_ca_controller,DEV_SEND_1905,...` now, are we?
Once it is ready, we'll refactor all tests and remove the BMLs which are not needed.

Reply:
@Galvien lets do that, we're not in a hurry to merge this, so if there's a better way lets go for it.